app/models/setting/puppet.rb

```diff
@@ -1,32 +1,12 @@
 class Setting::Puppet < Setting
-  def self.update_subnets_from_facts_types
-    {
-      'all' => _('All'),
-      'provisioning' => _('Provisioning only'),
-      'none' => _('None'),
-    }
-  end
-
   def self.default_settings
```

Comment:
Shouldn't the remaining settings also move to `config management` category, perhaps as a separate method that will be extracted to the plugin? (maybe with the exception of `update_environment_from_facts` which should go to facts)

Reply:
eeeeeeh, well probably, is it a proposal for moving Ansible settings to config management as well?

Reply:
probably at least those that relate to using ansible for config management (e.g. out of sync time etc), perhaps those that relate to ansible as a rex provider should be in the rex settings